cfme-performance/tests/ui_workloads/test_ui_all_vms.py

```diff
@@ -0,0 +1,98 @@
+"""Workload to stress WebUI with python requests."""
+from requests.auth import HTTPBasicAuth
+from utils.appliance import clean_appliance
+from utils.appliance import get_server_roles_ui_workload_all_vms
+from utils.appliance import set_server_roles_ui_workload_all_vms
+from utils.appliance import wait_for_miq_server_workers_started
+from utils.conf import cfme_performance
+from utils.grafana import get_scenario_dashboard_url
+from utils.log import logger
+from utils.providers import add_providers
+from utils.smem_memory_monitor import add_workload_quantifiers
+from utils.smem_memory_monitor import SmemMemoryMonitor
+from utils.ssh import SSHClient
+from utils.workloads import get_ui_all_vms_scenarios
+import pytest
+import requests
+import time
+
+
+@pytest.mark.usefixtures('generate_version_files', 'change_forgery_protection')
+@pytest.mark.parametrize('scenario', get_ui_all_vms_scenarios())
+def test_ui_all_vms(request, scenario):
+    """UI Workload to initiate navigations on the WebUI from python"""
+    from_ts = int(time.time() * 1000)
+    ssh_client = SSHClient()
+    logger.debug('Scenario: {}'.format(scenario['name']))
+
+    scenario_data = {'appliance_ip': cfme_performance['appliance']['ip_address'],
+        'appliance_name': cfme_performance['appliance']['appliance_name'],
+        'test_dir': 'ui-workload-all-vms',
+        'test_name': 'UI-All-VMs',
+        'appliance_roles': '',
+        'scenario': scenario}
+    quantifiers = {}
+    if scenario['clean_appliance']:
+        scenario_data['appliance_roles'] = get_server_roles_ui_workload_all_vms(separator=', ')
+    monitor_thread = SmemMemoryMonitor(SSHClient(), scenario_data)
+
+    def cleanup_workload(scenario, from_ts, quantifiers):
+        starttime = time.time()
+        to_ts = int(starttime * 1000)
+        g_url = get_scenario_dashboard_url(scenario, from_ts, to_ts)
+        logger.debug('Started cleaning up monitoring thread.')
+        monitor_thread.grafana_url = g_url
+        monitor_thread.signal = False
+        monitor_thread.join()
+        add_workload_quantifiers(quantifiers, scenario_data)
+        timediff = time.time() - starttime
+        logger.info('Finished cleaning up monitoring thread in {}'.format(timediff))
+    request.addfinalizer(lambda: cleanup_workload(scenario, from_ts, quantifiers))
+
+    if scenario['clean_appliance']:
+        wait_for_miq_server_workers_started(poll_interval=2)
+        clean_appliance(ssh_client)
+
+    monitor_thread.start()
+    wait_for_miq_server_workers_started(poll_interval=2)
```

Comment:
This wait for timeout in both situations I believe, I think we can just remove it.